__tests__/models.test.js

```diff
@@ -90,6 +90,49 @@ describe("/api/reviews", () => {
       });
   });
 });
+describe("/api/reviews/:review_id/comments", () => {
+  it("should receive a response of 200 and return all comments for the given review id", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toHaveLength(3);
+        expect(body.comments).toBeSorted();
```

Comment:
We need to ensure these are sorted by created_at with the most recent first so we need to drill a bit deeper with this assertion